components/lib/chip/__snapshots__/Chip.spec.js.snap

```diff
@@ -59,3 +59,33 @@ exports[`Chip check snapshots: label 1`] = `
   </div>
 </div>
 `;
+
+exports[`Chip when removable is true it returns with remove icon: after remove 1`] = `<div />`;
+
+exports[`Chip when removable is true it returns with remove icon: before remove 1`] = `
+<div>
+  <div
+    class="p-chip p-component"
+  >
+    <span
+      class="p-chip-remove-icon pi pi-times-circle"
+      tabindex="0"
+    />
+  </div>
+</div>
+`;
```

Comment:
You can observe the above how the `before remove` and `after remove` snapshots rendered different results (as expected)